.stylelintrc

```diff
@@ -0,0 +1,141 @@
+{
+  "plugins": [
+    "stylelint-scss"
+  ],
+  "rules": {
+    "color-hex-case": "lower",
```

Comment:
https://stylelint.io/user-guide/example-config/ lists all rules and options.
https://github.com/stylelint/stylelint-config-standard is a recommendation derived of several styleguides.
Stylelint Option "configOverrides" (https://stylelint.io/user-guide/node-api/) should it allow to use several configuration in parallel. Last configured rule is adopted, if a rule is set more than once.
Current .stylelintrc was built up from scratch. It would be interesting to compare it with https://github.com/stylelint/stylelint-config-standard